rfcs/20200902-pickle-for-keras.md

```diff
@@ -150,11 +150,9 @@ assert m1 == m2  # TODO: or some other check
 For `tf.keras.Model`, we can use `SaveModel` as the backend for `__reduce_ex__`:
 
 ``` python
-# tensorflow/python/.../training.py
 from tf.keras.models import load_model
 
-class Model:
-    ...
+class NewModel(Model):
```

Comment:
I'm curious as to why the name change? I was envisioning these code blocks as representing "here's a pseudocode of what this would look like if implemented in TF" and not necessarily "here's how users can create a picklable Model" which is the first thought that came to mind when I saw `NewModel(Model)`.